tags/info/floodgateskins.tag

```diff
@@ -0,0 +1,9 @@
+type: text
+aliases: skin, skins
+
+--
+
+Skins of Bedrock player should be visible to Java players on servers with [Floodgate 2.0](https://github.com/GeyserMC/Floodgate/wiki) installed.
+If they aren't, it's possible that the skin the player is using isn't supported (Such as Persona Skins with drastically different 3D models), or that the queue for skin uploading has grown too large (and you must simply wait).
+
+If you are using BungeeCord/Velocity: Installing Floodgate on the backend servers also makes it so that Bedrock player's don't have to switch backend servers for their skins to start displaying.
```

Comment:
```suggestion
If you are using BungeeCord/Velocity: Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
```